1209/9001/index.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: pid
+title: North Star Integrator
+owner: Combine Reality
+license: cc-bt-sa-4.0
+site: combinereality.org
+source: https://github.com/CombineReality/North-Star-Integrator ; 
+Based on lilyPadUSB-caterina, for use with the Combine Reality North Star Integrator board: https://github.com/CombineReality/ArduinoCore-avr
```

Comment:
Please only have one source site. It can either be software or hardware and its README should link to the other.